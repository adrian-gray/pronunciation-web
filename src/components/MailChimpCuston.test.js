import React from "react";
import { shallow } from "enzyme";
import { render, cleanup } from "@testing-library/react";

import MailChimpCustom from "./MailChimpCustom";

afterEach(cleanup);

test("MailChimpCustom", async () => {
  const props = {};

  const container = shallow(<MailChimpCustom {...props} />);

  expect(container.firstChild).toMatchSnapshot();
});
