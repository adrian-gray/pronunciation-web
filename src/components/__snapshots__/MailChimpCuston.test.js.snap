// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailChimpCustom 1`] = `undefined`;
